docusearch/templates/search/search.html

```diff
@@ -24,13 +24,24 @@
 
         <div>
         <dl>
-        <h3><a href="?q={{request.GET.q}}&amp;order_by=date_released">Order By</a> </h3>
+        <dd>
+            {% if request.GET.order_by == 'date_released' %}
+                <a href="?{{cleaned_query}}&amp;order_by=relevance ">Order By Date Relevance</a>
```

Comment:
Calling it cleaned query here isn't useful. What's it a link to? Changing the name will help this be self-documenting. I don't think is this a blocker for this pull request - but can we change it subsequently?